index.html

```diff
@@ -19,6 +19,8 @@
              <div class="menu">
                  <a href="#"><img src="images/Union.png" alt="menu icon"></a>
              </div>
```

Comment:
- [ ] kindly fix this design flaw with your toolbar section in your project as shown below it is important you do this before merging your project. :+1: 
![new-e](https://user-images.githubusercontent.com/89055502/235824906-747e3773-b933-41e0-b156-d0950fae4067.png)